### DISCUSSION TITLE
求助各位大佬：如何在鍵入聲母自成音節（m、n、ng）的候選字的同時，在候選欄中列出以m、n、ng開頭的其他含韻母的候選字？

### DISCUSSION THREAD

**Discussion by ghost** [rime/home#1636] [Q&A] [answered]
各位大佬好~以下是我現有的代碼：
- erase/^xx$/
    - erase/^0$/
    - "xform/'//"
    - "derive/1/q/"
    - "derive/2/x/"
    - "derive/3/r/"
    - "derive/^([A-Za-z]+)([nghk]{0,2})([1-3qxr]{0,2})(\\{.*\\})?$/$1$2/"  
    - abbrev/^([a-z]).+$/$1/
    - abbrev/^([ptkgszc]h).+$/$1/
    - abbrev/\b(tsh)\w*.+$/$1/
    - abbrev/\b(ts)[^\Wh]*$/$1/
    - abbrev/^(dz|ng).+$/$1/

根據上述代碼，在鍵入m時，只能匹配到音碼爲m1，m2，m3的候選字，而無法像搜狗的粵語拼音輸入灋那樣，在這些聲母自成音節的候選字後再列出以m爲聲母，同時含有韻母的候選字。

**Comment by LEOYoon-Tsaw** [comment, marked as answer]
可以試試把abbrev換成derive

**Reply by ghost** [reply]
成功啦，多謝您！
